app/controllers/taxons_controller.rb

```diff
@@ -48,6 +48,18 @@ def update
     redirect_to taxons_path
   end
 
+  def destroy
+    resp = Services.publishing_api.unpublish(params[:id], type: "gone")
+
+    message = if resp.code == 200
```

Comment:
Pretty sure you'll get the linter complaining here about the alignment and indentation of the if block.

You can resolve this by either wrapping the whole thing in a begin/end, or by lining up the `else` and `end` with the `if`.  I think the latter is ugly as hell but either would do the job.